src/lib/utils.ts

```diff
@@ -213,3 +217,12 @@ export function countQuestion(arr: IResponseGetQuestionPagination[]) {
 
   return totalQuestion
 }
+export function countPublicUser(arr: IResponseGetPublicUserList[]) {
```

Comment:
How about make it more generic?

Rename the fn name to `calculatePageItemCount`

Reply:
done mas